### DISCUSSION TITLE
Publishing Vercel Analytics snapshots

### DISCUSSION THREAD

**Discussion by shandysulen** [vercel/community#1458] [Idea]
I'm interested in the idea of publishing/sharing Visitors and Page View data with users. It would be neat if Vercel offered a way to make this data public on an opt-in basis, perhaps exposing the data through a reserved serverless endpoint.

**Comment by amyegan** [comment]
Thanks for sharing this idea! The team has plans to add an Audiences API, which would allow you to use and share the data however you see fit. 😁 It's being discussed in https://github.com/vercel/community/discussions/1068

**Reply by shandysulen** [reply]
Awesome! Feel free to close/end this discussion if possible :)
